microshift_running_apps/microshift-embedded-apps-on-rhel-edge.adoc

```diff
@@ -9,10 +9,10 @@ toc::[]
 You can embed microservices-based workloads and applications in a {op-system-ostree-first} image to run in a {microshift-short} cluster. Embedded applications can be installed directly on edge devices to run in air-gapped, disconnected, or offline environments.
 
 [id="microshift-add-app-RPMs-to-rpm-ostree-image_{context}"]
-== Adding application RPMs to an rpm-ostree image
+== Adding application RPMs to a rpm-ostree image
```

Comment:
This should stay "an", because when you read this, you read it as r-p-m (starts with the sound "are")